### PR TITLE
Update to version 11.0 for build-impacting version numbers (and reduce hard-coded references where possible)

### PR DESCRIPTION
**Pull request by mcdurdin**
This PR prepares the 11.0 alpha branch for build and CI

### CODE REVIEW

File: windows/src/global/delphi/general/KeymanVersion.pas [hunk 1]
```diff
@@ -20,14 +20,15 @@
...
   SKeymanVersion90 = '9.0';
   SKeymanVersion80 = '8.0';
   SKeymanVersion70 = '7.0';
   SKeymanVersion60 = '6.0';
   SKeymanVersion50 = '5.0';
 
-  SKeymanVersion = SKeymanVersion100;
+  SKeymanVersion = SKeymanVersion110;
 
   SKeymanKeyboardVersion = SKeymanVersion100;
```
Comment (mcdurdin) [inline windows/src/global/delphi/general/KeymanVersion.pas]: The keyboard version stays at 10.0 until there is a reason to bump it to 11.0.

File: windows/src/Defines.mak [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # DEBUG=1
 
+KEYMAN_VERSION=11.0
```
Comment (jahorton) [inline windows/src/Defines.mak]: So we are not drawing this from VERSION.md?

Comment (jahorton) [reply windows/src/Defines.mak]: Looks like there are a few other places in the Windows code also ignoring it.

Comment (mcdurdin) [reply windows/src/Defines.mak]: We can't at present. We'd have to build a file to read it. Not worth the effort.

Comment (mcdurdin) [reply windows/src/Defines.mak]: Also, the Windows files need deliberation w.r.t. the keyboard versioning as well as product versioning. Better to document these in the version upgrade docs I think than to automate.

**Pull request merged**